packages/calcite-components/src/components/fab/fab.scss

```diff
@@ -1,16 +1,47 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-fab-background-color: Specifies the component's background color.
+ * @prop --calcite-fab-shadow: Specifies the component's shadow.
+ * @prop --calcite-fab-shadow-hover: Specifies the component's shadow when hovered.
+ * @prop --calcite-fab-shadow-press: Specifies the component's shadow when pressed.
+ */
+
 :host {
-  @apply flex bg-transparent;
+  @apply flex;
+
+  background-color: var(--calcite-fab-background-color, transparent);
 }
 
 @include disabled();
 
 calcite-button {
-  @apply shadow-2;
+  --tw-shadow: var(--calcite-fab-shadow, 0 6px 20px -4px rgba(0, 0, 0, 0.1), 0 4px 12px -2px rgba(0, 0, 0, 0.08));
+  --tw-shadow-colored: var(
+    --calcite-fab-shadow,
+    0 6px 20px -4px var(--tw-shadow-color),
+    0 4px 12px -2px var(--tw-shadow-color)
+  );
+  box-shadow: var(--tw-ring-offset-shadow, 0 0 #0000), var(--tw-ring-shadow, 0 0 #0000), var(--tw-shadow);
   &:hover {
```

Comment:
I’m trying to get rid of the tw values to set a shadow token.
 
What happens when `--tw-ring-offset-shadow` goes away here:
`box-shadow: var(--tw-ring-offset-shadow, 0 0 #0000), var(--tw-ring-shadow, 0 0 #0000), var(--tw-shadow);`
 
I could sub the `--tw-shadow` with: `var(--calcite-fab-shadow, 0 6px 20px -4px rgba(0, 0, 0, 0.1), 0 4px 12px -2px rgba(0, 0, 0, 0.08));`
Not sure what to do with the ring values.

@ashetland @SkyeSeitz @macandcheese

Reply:
It seems related but may be out of scope for this issue. And it doesn't really answer what to use for ring values. These are the first two values on box shadow and are interpreted as <offset-x> and <offset-y>.

Reply:
Oh gotcha. Your sub values look correct to me and match the medium shadow we have spec'ed for FAB in #10050 so we wouldn't have any visual changes when that one gets picked up.